src/Pages/Signup/Signup.js

```diff
@@ -0,0 +1,347 @@
+        identity: this.state.identity,
+        password: this.state.password,
+        name: this.state.name,
+        phone: this.state.phone,
+        email: this.state.email,
+        address: this.state.address,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => {
+        console.log('결과: ', result);
+        if (result.message === 'SUCCESS') {
+          alert('가입을 축하드립니다.');
+          this.props.history.push('/login');
+        } else if (result.message === 'IDENTITY_ALREADY_EXISTS') {
+          this.setState({ isAvailedId: '입력값을 확인해주세요' });
+        }
+      });
+  };
+
+  render() {
+    const { isDaumPost, firstAddress } = this.state;
+
+    //id체크
+    const idreg = /^[a-z0-9]{5,15}/g;
+    const isIdValid = idreg.test(this.state.identity);
+
+    return (
+      <div>
+        <nav className="navi">
+          <Header />
+        </nav>
+        <div className="contents_All">
+          <div className="img">
+            <img src="/images/adlogo.png" alt="광고사진" className="a_Img" />
+          </div>
+          <article>
+            <div className="head">
+              <header>오아시스 회원 가입</header>
+              <span className="descrip">
+                세상을 바꾸겠다는 거창한 철학보다는 한 사람의 작은 바람을 담은
+                소비가 가족을 행복하게 하고, 행복한 가족이 모여 행복한 사회를
+                만들 수 있다는 단순하고 소박한 가치를 실천합니다.
+              </span>
+            </div>
+            <form className="sing_up">
+              <div className="inputBox_Id">
+                <input
+                  id="identity"
+                  type="text"
+                  name="identity"
+                  onChange={this.handleInputChange}
+                  placeholder="아이디 (7~12 자 , Only 영문 숫자 조합)"
+                />
+                <button
+                  className="btnId"
+                  style={{
+                    backgroundColor: isIdValid ? '#6ca437' : '#d2f7d2',
+                  }}
+                  disabled={!isIdValid}
+                  onClick={this.checkDuplicateId}
+                >
+                  중복체크
+                </button>
+              </div>
+
+              {!validateId(this.state.identity) ? (
+                <p className="errMsg">아이디 형식이 올바르지 않습니다.</p>
+              ) : (
+                <p className="errMsg"></p>
+              )}
+
+              <div className="inputBox_Pw">
+                <input
+                  id="password"
+                  name="password"
+                  onBlur={this.pwCheck}
+                  onChange={this.handleConfirmPassword}
+                  type="password"
+                  placeholder="비밀번호 (8~18 자, 대소문자 숫자 특수문자 모두 포함)"
+                />
+
+                <input
+                  id="rePassword"
+                  name="rePassword"
+                  onChange={this.handleConfirmPassword}
+                  type="password"
+                  placeholder="비밀번호확인"
+                />
+                <p className="errMsg">{this.state.isAvailedPassword}</p>
```

Comment:
- 이 부분도 바꿔주세요! 아마 아래와 같은 그림이 되어야 합니다!
```suggestion
                <p className="errMsg">
                  {confirmPassword(password, rePassword) ? '' : '비밀번호가 일치하지 않습니다'}
                </p>
```